### ISSUE TITLE
Preserve the load order on the pages list in Calypso when there are more then 50 pages.

### ISSUE THREAD

**Issue by KirkwallDay** [Automattic/wp-calypso#68121]
### What

Currently, when the page count exceeds 50 pages, the load order in Calypso changes to last edited post only. Before that, it will show parent pages above their child pages, after it will not. Also, it adds a - at the beginning of the page title, this should also be removed.

Before 51 pages:
![Screen Shot 2022-09-21 at 12 38 59 PM](https://user-images.githubusercontent.com/49131029/191584582-18f37160-516a-45c9-af55-8a36a648061a.png)

After 51 pages: 
![Screen Shot 2022-09-21 at 12 40 36 PM](https://user-images.githubusercontent.com/49131029/191584812-bbd64e77-885f-480c-9800-f8254cf382ab.png)

Gallery has moved much further down the page list, and now has a - to indicate it's a child page, but it's parent is much further up the list now.

Two customer reports:

- [ ] 5563725-zen
- [ ] 5575173-zen 

### Why

This can be confusing for users to determine which pages have children, and also when they cross the 51 page threshold the view suddenly changes and is also a confusing experience.

### How

It would be best if we can keep the old view, there is discussion about this, in this issue: https://github.com/Automattic/wp-calypso/issues/60280#issuecomment-1136813962

And in this slack thread: p1663378153241789-slack-C03TY6J1A

**Comment by Greatdane** [comment]
📌 FINDINGS/SCREENSHOTS/VIDEO
- Almost tempted to label this as a bug, as I am not sure this is intended behavior. I will let Quality Squad determine the best way forward on that! 

📌 ACTIONS
- Marked as Triaged for Quality Squad review
- Feature request kept

**Comment by youbrokesomething** [comment]
From the linked ticket, it looks like this is the currently expected behavior. More than 50 pages will result in a Chronological view. So, I'm going to keep this as a feature request and move it to that queue.

**Comment by hacchism** [comment]
another report in 5648190-zen, suggested using classic view as a workaround

**Comment by rw-ye** [comment]
5846271-zd-woothemes

Suggested classic view workaround.

**Comment by marianearchimbaud** [comment]
5920484-zd another user has reported the same issue

**Comment by sudeepbaral** [comment]
Another report here:

- [ ] 6428692-zen

Suggested using Classic View as a workaround

**Comment by ccwalburn** [comment]
Another report here:

- [ ] 6538200-zen

**Comment by kelasante** [comment]
Another report here:

6569256-zen

**Comment by tellisbethel** [comment]
Another report here: 6527845-zd-a8c
Suggested Classic Editor workaround

**Comment by CodeAllNightNDay** [comment]
Report here: 6648395-zen

**Comment by wpshellbelle** [comment]
Another report here: 6710916-zen

**Comment by DustinHartzler** [comment]
Another report in 6782614-zen

**Comment by inaikem** [comment]
@wojtekn 👋 We had a similar issue (presentation-wise) here: https://github.com/Automattic/wp-calypso/issues/60280

Is the above related in some way?

**Comment by wojtekn** [comment]
@inaikem yes, those two tickets are related. To give some summary - before https://github.com/Automattic/wp-calypso/issues/60280 the behavior was confusing to users:
- on the initial load, the page showed less than 20 posts in “chronological” mode
- after scrolling to the bottom, infinite load caused loading more posts, and when they exceeded 20, the view switched to "hierarchical" mode

The most comprehensive fix would require changing the feature architecture completely, as we would need to move hierarchy/indent calculation from Node code to the API endpoint. After a long discussion, we decided to solve the user experience gap in the following way:

- the user who has less than 50 pages will always see a hierarchical view
- the user who has more than 50 pages will always see a chronological view

**Comment by gabriel-fuentes** [comment]
7203893-zen

**Comment by caitlindawn** [comment]
7429662-zen

**Comment by OmarFPG** [comment]
7429662-zen asking for an update again.

**Comment by philnick206** [comment]
7462298-zd-a8c requesting an update again.

**Comment by wojtekn** [comment]
@OmarFPG @philnick206 there is no immediate plan to change this behavior and the current state looks as follows:

> After a long discussion, we decided to solve the user experience gap in the following way:
> 
> - the user who has less than 50 pages will always see a hierarchical view
> - the user who has more than 50 pages will always see a chronological view

**Comment by OmarFPG** [comment]
7704805-zen

**Comment by Greatdane** [comment]
7885655-zd-a8c 

The user is confused as to why the page layout would change after 50 pages.

Can we rethink this setup? It seems strange to change the layout after a certain number of pages.

**Comment by ezebecke** [comment]
Also on 9247734-zd-a8c

This change of order is very confusing for users and for HEs trying to troubleshoot and understand what happened.

**Comment by inaikem** [comment]
@Copons, I know this issue doesn't quite fall into your group's area but I'd love some help increasing visibility to get it resolved.

The current logic of breaking the page load order when the page count exceeds 50 makes little logical sense to me.

**Comment by wojtekn** [comment]
@inaikem is it still an issue after untangling https://github.com/Automattic/wp-calypso/issues/95621 ?

**Comment by Copons** [comment]
As suggested by @wojtekn, the Pages screen is being untangled (this issue specifically https://github.com/Automattic/wp-calypso/issues/95620), so this issue is about to become irrelevant soon.

Starting this week, 90% of users will only get the wp-admin Pages screen (among other screens). This issue will remain for the other 10% until the end of the experiment (no ETA), but arguably the priority and usefulness of fixing it will be almost insignificant.

**Comment by inaikem** [comment]
@wojtekn, @Copons, thank you! I'll close this issue for now and loop back in a couple of weeks to review the status of the experiment 🙏
